internal/pkg/util/fs/helper_linux_test.go

```diff
@@ -311,17 +311,17 @@ func TestEvalRelative(t *testing.T) {
 	}{
 		{"/bin", "/usr/bin"},
 		{"/sbin", "/usr/sbin"},
-		{"/bin/bin", "/usr/bin"},
```

Comment:
I'm pretty confused here. This looks like it used to resolve down to an actual directory `/usr/bin` but now the result `/bin` still includes a symlink?

Reply:
Previous version was broken and resolved links outside of the relative root

Reply:
Sorry I'm not clear what "relative root" means? Do you mean that if the `root` contains a symlink then only symlinks in `path` that are *not* in `root` are resolved?

Reply:
See explanations below